manager/src/main/java/org/apache/hertzbeat/manager/service/PluginService.java

```diff
@@ -58,7 +63,7 @@ public interface PluginService {
      * @param execute run plugin logic
      * @param <T> plugin type
      */
-    <T> void pluginExecute(Class<T> clazz, Consumer<T> execute);
+    <T> void pluginExecute(Class<T> clazz, Consumer<T> execute, BiConsumer<T, List<Configmap>> biConsumer);
```

Comment:
It is recommended to wrap parameters in objects to improve the scalability of the plugin interface and facilitate the subsequent transfer of other data to the plugin for use.